src/Microsoft.DotNet.Interactive.ExtensionLab.Tests/MsSqlConnectionTests.cs

```diff
@@ -21,19 +22,28 @@ public MsSqlConnectionTests(ITestOutputHelper output)
             _output = output;
         }
 
-        [MsSqlFact("Persist Security Info=False; Integrated Security=true; Initial Catalog=AdventureWorks2019; Server=localhost", Skip = "not ready")]
-        public async Task It_can_connect_and_query_data()
+        private async Task<CompositeKernel> CreateKernel()
         {
-            using var kernel = new CompositeKernel
+            var csharpKernel = new CSharpKernel().UseNugetDirective();
+            await csharpKernel.SubmitCodeAsync(@$"
+#r ""nuget:runtime.osx-x64.native.microsoft.sqltoolsservice,3.0.0-release.52""
```

Comment:
@jonsequitur - I've hard coded the SqlToolsService nuget reference for the time being. Have any ideas what is the best way to handle this? I was thinking, based on the running platform, I could try and resolve the platform reference.

Reply:
You don't need to reference the platform-specific packages directly. Reference Microsoft.SqlToolsService and the appropriate platform-specific package will be installed transitively.

Reply:
I see. Thanks for the tip!